Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+@app.route('/add/<int:num1>/<int:num2>')
+def add(num1, num2):
+	return "{}".format(num1 + num2)
```

Comment:
good!

Optionally, as of Python 3.6 you can also use f-strings like so:

```python
f"{num1 + num2}"
```

which is slightly more concise 😉